### PR TITLE
Fresher content

### PR DESCRIPTION
**Pull request by adgad**
/cc @wheresrhys @phamann @Lrnk 

Fetches the full week's content every 10 minutes, but tops that up with the last day's content fetch which happens every minute.

Should hopefully mean content is fresher and reduce the 'New article in your myFT' pushes.

### CODE REVIEW

File: server/content/capi1-articles.js [hunk 1]
```diff
@@ -32,13 +33,16 @@ Object.getOwnPropertyNames(EventEmitter.prototype).forEach(function (fn) {
...
-Content.prototype.fetch = function() {
+Content.prototype.fetch = function(timeIntervals) {
 	var previousArticles = this.articles.reduce(function (map, a) {
 		map[a.item.id] = true;
 		return map;
 	}, {});
 	var baseTime = new Date().getTime();
-	var fetches = [0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5, 5.5, 6, 6.5].map(function (i) {
+
+	timeIntervals = timeIntervals || [0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5, 5.5, 6, 6.5];
```
Comment (phamann) [inline server/content/capi1-articles.js]: Surely we could create this list functionally: https://lodash.com/docs#range

**Comment by phamann** [comment]
:+1:

File: server/content/capi1-articles.js (hunk 1 above)
Comment (wheresrhys) [inline server/content/capi1-articles.js]: `Array((2*7) + 1).join('.').split('').map(function (item, i) {return i/2})`

yuk though.

**Comment by adgad** [comment]
Now sorting and only firing event on new content

**Pull request merged**